src/lib.rs

```diff
@@ -85,6 +85,7 @@
 //! | --- | --- | --- | --- |
 //! | `std::sync::Mutex` | [`BusManagerStd`] | [`new_std!()`] | `std` |
 //! | `cortex_m::interrupt::Mutex` | [`BusManagerCortexM`] | [`new_cortexm!()`] | `cortex-m` |
+//! | `spin::Mutex` | [`BusManagerXtensa`] | [`new_xtensa!()`] | `xtensa` |
```

Comment:
To be pedantic, this should now be a `shared_bus::XtensaMutex`.  It is still worth mentioning the source type though, so I'd make this

```suggestion
//! | `shared_bus::XtensaMutex` (`spin::Mutex` in critical section) | [`BusManagerXtensa`] | [`new_xtensa!()`] | `xtensa` |
```